app/models/collection.rb

```diff
@@ -66,6 +66,10 @@ def show_to?(user)
     (!self.restricted && self.works.present?) || (user && user.like_owner?(self)) || (user && user.collaborator?(self))
   end
 
+  def self.by_user(user)
+    Collection.includes(:owners).where(owner_user_id: user.id).joins(:collaborators)
```

Comment:
Okay, there are two different N:N mappings between `users` and `collections`.  This one should only look at collaborators, so `Collection.joins(collaborators).where("collection_collaborations.user_id = ?", user.id)` is probably the query we want.

Reply:
I may be wrong, but I think there's a nice shorthand for this on the user model
`user.collection_collaborations`

Reply:
@benwbrum @jackweinbender I tried both of your suggestions, but the tests continue to fail.